features/GettingStarted.md

```diff
@@ -29,15 +29,15 @@ the files in the `spec` directory to RSpec.
 
 ### Run migrations
 
-    $ rake db:migrate && rake db:test:prepare
+    $ rails db:migrate && rails db:test:prepare
 
 ### Run RSpec
 
     $ rake spec
 
 or
 
-    $ rspec spec --format documentation
```

Comment:
Doc format is the default when running just one file.
`spec` is the default directory.

Reply:
Explicit is better :/